main.rkt

```diff
@@ -514,9 +514,14 @@
           (if (bfodd? (ival-lo-val y))
               (ival-neg (ival-pow-pos (ival-exact-fabs x) y)) ; Use fabs in case of [x, 0]
               (ival-pow-pos (ival-exact-fabs x) y))
-          ; If y is non-integer point interval, it must be an even
-          ; fraction (because all bigfloats are) so we always error
-          ival-illegal)
+
+          (if (bfnegative? (ival-lo-val y))
+            ival-illegal
+            (let ([frac (rationalize (bigfloat->rational (ival-lo-val y)) (expt 2 (- (bf-precision))))])
```

Comment:
This solution will not work.
What it does is finding the simplest form of a fraction using float representation.
The problem arises when `y` can not be represented by a division of floats due to its large significand (higher precision).
This is, probably, the reason why IEEE considers this case as illegal, indeed, as it was said previously, all the fractions are even in this standard.